README.md

```diff
@@ -32,6 +32,7 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
 - Install [Docker](https://docs.docker.com/get-docker/) (remember to complete the [Linux post-install steps](https://docs.docker.com/engine/install/linux-postinstall/) if you're on Linux)
 - Install docker-compose. The best way to get an up-to-date version is to get it from their [releases page](https://github.com/docker/compose/releases)
 - Navigate into the repository's main directory (`worldcubeassociation.org`)
+- Note if you are running an `ARM` based computer like the M1 Macbooks, you will need add the following to your mysql image installation: `platform: linux/amd64` after [this line](https://github.com/thewca/worldcubeassociation.org/blob/master/docker-compose.yml#L33)
```

Comment:
Nitpick: need **to** add the following

Not sure because I'm not a native English speaker